packages/Skeleton/SkeletonProvider.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import AutoSkeletonizer from './AutoSkeletonizer'
+
+export const SkeletonContext = React.createContext(false)
```

Comment:
can we replace context-api with recompose instead to support older versions of react. Thanks :)

Reply:
done